packages/windows/download_appveyor_build.rb

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env ruby
+
+require "pathname"
+require "veyor"
+require "octokit"
+
+def env_value(name)
+  value = ENV[name]
+  raise "Specify #{name} environment variable" if value.nil?
+  value
+end
+
+client = Octokit::Client.new(:access_token => env_value("GITHUB_TOKEN"))
```

Comment:
GitHub token is needless because `groonga/groonga` is a public repository.

Reply:
Oh, surely no need to use GITHUB_TOKEN. I'll fix.

Reply:
removed.